content/blog/what_is_err_name_not_resolver_error_and_how_to_fix_It.md

```diff
@@ -0,0 +1,231 @@
+
+2. **DNS Server Issues**: If your DNS server is temporarily down, slow, or experiencing technical issues, it cannot process DNS requests efficiently, resulting in resolution failures and the `ERR_NAME_NOT_RESOLVED` error.
+
+3. **DNS Cache Problems**: Your operating system and browser store previously resolved domain names to speed up future access. However, this cache can become outdated or corrupted, causing the DNS to resolve to incorrect IP addresses and leading to errors.
+
+4. **Misconfigured Network Settings**: Incorrect or outdated DNS server settings on your device can disrupt the proper functioning of DNS queries. If your system attempts to connect to an inactive or misconfigured DNS server, you will encounter resolution problems.
+
+5. **Firewall or Antivirus Restrictions**: Security software such as firewalls, antivirus, or network security tools can inadvertently block DNS queries. This usually happens when these tools are overly restrictive or misconfigured, mistakenly identifying legitimate DNS requests as harmful activities, thus blocking access to certain websites.
+
+### Step-by-Step Solutions to Fix ERR_NAME_NOT_RESOLVED
+
+#### 1. Double Check the URL
+Before diving into technical fixes, ensure the website address you’ve typed is correct. A simple typo in the domain name or an extra space can cause the browser to fail in resolving the site. Even an unnoticed mistake like `.co` instead of `.com` or a misplaced character can trigger this error.
+
+**Solution**: Carefully check the URL for any errors or unnecessary spaces. If possible, try accessing the website using a different device to verify that the issue is not due to a typo.
+
+{{< image "what_is_err_name_not_resolver_error_and_how_to_fix_It/pinggy_site_ss.webp" "Url Check ScreenShot" >}}
+
+#### 2. **Verify DNS Resolution**  
+   - Check if DNS resolution is functioning correctly by querying it from the command line:
+
+     **Windows (Command Prompt or PowerShell):**
+     - Using `nslookup`:
+       ```sh
+       nslookup example.com
+       ```
+     - Using `ping` (for basic resolution):
+       ```sh
+       ping example.com
+       ```
+     - Using `Resolve-DnsName` (PowerShell only):
+       ```powershell
+       Resolve-DnsName example.com
+       ```
+
+     **Linux (Terminal):**
+     - Using `dig` (most detailed):
+       ```sh
+       dig example.com
+       ```
+     - Using `nslookup`:
+       ```sh
+       nslookup example.com
+       ```
+     - Using `host`:
+       ```sh
+       host example.com
+       ```
+     - Using `ping` (for basic resolution):
+       ```sh
+       ping example.com
+       ```
+
+   These commands help confirm whether DNS resolution is functioning correctly and can identify if the issue lies with DNS settings or connectivity.
```

Comment:
In one or two lines write what is the output if everything is correct, and what is the output if dns lookup fails.